### PR TITLE
[fix] Check None value for Pytorch model gradients histogram

### LINKED ISSUE #1925: Pytorch track_gradients_dists errors out if some parameters don't have gradients
## 🐛 Bug

When collecting gradients for each layer weight of a model, the function `get_model_layers` errors out if some model parameters don't have gradients.

### Expected behavior

Ignore weights if grad is None.

### Environment

- Aim Version (e.g., 3.11.1)
- Python version 3.10
- pip version 22.0
- Any OS

### PR DESCRIPTION
**Pull request by kage08**
When tracking histograms of gradients, some tensors may not have gradients during specific training loops.
So check if grad is not None before converting to numpy.

This commit fixes #1925

### CODE REVIEW

**Comment by gorarakelyan** [comment]
@kage08 thanks a lot for the contribution! 🙌 
@alberttorosyan could you please take a look at this?

**Comment by SGevorg** [comment]
@kage08 thanks a lot for the contribution. 🙏 
Could you open a corresponding issue (with some usage context) and connect to this PR?
We use issues to track the changes and by adding to milestones to which version the changes got shipped with 🙌

File: aim/sdk/adapters/pytorch.py [hunk 1]
```diff
@@ -62,12 +62,14 @@ def get_model_layers(model, dt, parent_name=None):
             layers[layer_name] = {}
             if hasattr(m, 'weight') \
                     and m.weight is not None \
-                    and hasattr(m.weight, dt):
+                    and hasattr(m.weight, dt) \
```
Comment (alberttorosyan) [inline aim/sdk/adapters/pytorch.py:65]: can lines 65-66 be rewritten as `getattr(m.weight, dt, None) is not None`? 🤔

File: aim/sdk/adapters/pytorch.py [hunk 2]
```diff
@@ -62,12 +62,14 @@ def get_model_layers(model, dt, parent_name=None):
...
+                    and getattr(m.weight, dt) is not None:
                 layers[layer_name]['weight'] = get_pt_tensor(getattr(m.weight, dt)).numpy()
 
             if hasattr(m, 'bias') \
                     and m.bias is not None \
-                    and hasattr(m.bias, dt):
+                    and hasattr(m.bias, dt) \
```
Comment (alberttorosyan) [inline aim/sdk/adapters/pytorch.py:71]: same here

**Review by alberttorosyan — CHANGES_REQUESTED** [review]
@kage08 thanks for contribution 🙌 
may I ask you to update the CHANGELOG.md to include the fix in `Unreleased` section?
also I've changed the PR header to match the naming conventions.

File: aim/sdk/adapters/pytorch.py (hunk 1 above)
Comment (kage08) [reply aim/sdk/adapters/pytorch.py]: Yup. Committed the requested changes.

**Review by alberttorosyan — CHANGES_REQUESTED** [review]
looks good!
the remaining steps:
1. open a GH issue
2. update CHANGELOG.md to include this fix

**Comment by gorarakelyan** [comment]
Made a minor fix in the changelog :) 

btw, it is worth considering getting the weight tensor only once, isn't it?

e.g:
```py
weight = None
if hasattr(m, 'weight') and m.weight is not None:
    weight = getattr(m.weight, dt, None)

if weight is not None:
    layers[layer_name]['weight'] = get_pt_tensor(getattr(m.weight, dt)).numpy()
```

instead of:
```py
if hasattr(m, 'weight') and m.weight is not None and getattr(m.weight, dt, None) is not None:
    layers[layer_name]['weight'] = get_pt_tensor(getattr(m.weight, dt)).numpy()
```

thoughts? @kage08 @alberttorosyan

**Review by gorarakelyan — APPROVED** [review]
@kage08 looks awesome 🙌
